services/web/server/src/simcore_service_webserver/studies_dispatcher/_catalog.py

```diff
@@ -62,6 +62,9 @@ async def iter_latest_osparc_services(
     engine: Engine = get_database_engine(app)
     settings: StudiesDispatcherSettings = get_plugin_settings(app)
 
+    # Select query for latest version of the service
+    latest_view = create_select_latest_services_query().alias("latest_view")
```

Comment:
I would stick with the previous name, this way we do not confuse people. A view is not a query in SQL language
```suggestion
    latest_services = create_select_latest_services_query().alias("latest_view")
```